bootstrap.md

```diff
@@ -0,0 +1,136 @@
+# Filecoin Bootstrapping Routine
+
+This spec describes how to implement the Filecoin bootstrapping protocol, for related systems, see:
+
+- [Sync](sync.md) on how Filecoin nodes sync with bootstrap nodes to catch up to the chain and stay synced after the initial bootstrapping has occured.
+- [Network Protocols](./network-protocols.md) on how Filecoin nodes can communicate with each other, with for instance, an [initial handshake](./network-protocols.md#hello-handshake), or [block syncing](./network-protocols.md#blocksync).
+- [Operation](./operation.md) on various operations a functional Filecoin node needs to run, like [DHT routing](./operation.md#dht-for-peer-routing.md).
+
+## What is bootstrapping in Filecoin?
+
+When a Filecoin node first comes online, it must find the latest head for the chain and then initiate the syncing routine in order to catch up to the chain state and maintain it thereafter as new blocks come in. This is covered in the [sync spec](./sync.md). Bootstrapping is the process through which a node finds the chain's latest head in the first place (thereby enabling it to sync), through other peers in the network.
+
+At its simplest, bootstrapping can be thought of as secure network joining. It must enable a new node to get the latest head/chain that the majority of the network power is mining off and must be both:
+
+- Robust — able to do so in adversarial networks (so long as honest nodes make up the majority of the network).
+
+- Consistent — able to reliably get the canonical chain (even given poor latency or state loss in our initial peer set).
```

Comment:
what do you mean by state loss?

Reply:
I was trying to bucket all other non-byzantine reasons other than latency why some node may not be up-to-date...

Better way to phrase?

Reply:
eh, i think thats okay. maybe could specify 'non byzantine state loss' ?